src/script/components/loading-button.ts

```diff
@@ -14,6 +14,10 @@ export class LoadingButton extends LitElement {
         width: 1.8em;
       }
 
+      fast-progress-ring::part(indeterminate-indicator-1) {
+        stroke: var(--primary-purple);
```

Comment:
controllable via css variables on the component, easier than trying to find the part imo

--accent-foreground-rest
--neutral-fill-rest